django_server/.env.sample

```
@@ -0,0 +1,80 @@
+
+# Session Configuration
+SESSION_COOKIE_AGE=86400  # 24 hours in seconds
+SESSION_COOKIE_SECURE=False  # Set to True in production with HTTPS
+SESSION_COOKIE_HTTPONLY=True
+
+# Email Configuration (for password reset)
+EMAIL_BACKEND=django.core.mail.backends.console.EmailBackend
+EMAIL_HOST=smtp.gmail.com
+EMAIL_PORT=587
+EMAIL_USE_TLS=True
+EMAIL_HOST_USER=your-email@gmail.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EMAIL_HOST_USER key should go before the EMAIL_PORT key

(UnorderedKey)